vta/src/de10nano/de10nano_mgr.h

```diff
@@ -0,0 +1,554 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ * \file de10nano_mgr.h
+ * \brief DE10-Nano fpga manager.
+ */
+
+#ifndef VTA_DE10NANO_DE10NANO_MGR_H_
+#define VTA_DE10NANO_DE10NANO_MGR_H_
+
+extern "C" {
```

Comment:
I suggest taking the following format

```c
#ifdef __cplusplus
extern "C" {
#endif 

// ...

#ifdef __cplusplus
}
#endif 
```

In addition, I think this is a C++ header file (please correct me if I understood), it's better to include C++ variant of the standard library, like `#include <cstdint>`

Reply:
This style is kind of nowadays redundant as the compiler knows already what to do.
You are right, this is a C++ header file and chances are it will not be used in a C only context.
I can make it pure C++ if you like.